### ISSUE TITLE
PIL Imcompatible

### ISSUE THREAD

**Issue by csyhhu** [tristandeleu/pytorch-meta#23]
Hi!

Thanks for your code! I found a imcompatible problem with PIL. 
I install torchmeta(1.3.0), it upgrade my torch to 1.4 and torchvision to 0.5. Then when I run the demo code, it comes up with the error like:

```
im = new(self.mode, size, fillcolor)
  File "/home/csy/anaconda3/envs/csy_env/lib/python3.7/site-packages/PIL/Image.py", line 2505, in new
    return im._new(core.fill(mode, size, color))
TypeError: function takes exactly 1 argument (3 given)

```
Then I found in https://github.com/pytorch/vision/issues/1759, where it seems that torchvision (0.5) has some imcompatible problem with PIL. So I downgrade torchvision to 0.4 (it automatically downgrade my torch to 1.2), and it works.

However, as you mention, torchmeta is not compatible with torch == 1.2.0.

But the demo code can work on the combination of these verision: torch (1.2.0), torchvision (0.4.0), PIL (6.2.1)

Looking forward to any other solution.

Best regards,
Shangyu

**Comment by tristandeleu** [comment]
Hi Shangyu, thank you for the notice! I froze the version of PIL because it was conflicting on an earlier version of torchvision but they have probably fixed it since then so it's probably a PIL version issue (I think torchvision 0.4/PIL < 7.0 or torchvision 0.5/PIL 7.0). I will test it further and update that!

**Comment by tristandeleu** [comment]
This should be fixed now in version 1.3.2. I have temporarily patched the rotate function with the fix suggested in https://github.com/pytorch/vision/issues/1759#issuecomment-583826810, while waiting for a new release of torchvision.
